server/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -1267,7 +1267,7 @@ public void onTimeout(TimeValue timeout) {
             }
         }
 
-        logger.info("started");
+        logger.info("started {}", transportService.getLocalNode());
```

Comment:
This logs something like `started {runTask-0}{xxhYOWXfQdyqY2Au33UKmA}{XN9z8jtRT5OiRzxJxDqFFg}{127.0.0.1}{127.0.0.1:9300}{cdfhilmrstw}{ml.machine_memory=16557490176, ml.max_jvm_size=536870912, testattr=test, xpack.installed=true}` :+1: 

Is it also worth to update https://github.com/elastic/elasticsearch/blob/327163d07e431bdcd2ba26f2e65fba8df3b47481/server/src/main/java/org/elasticsearch/node/Node.java#L320-L321

to log the exact build hash if the version is a SNAPSHOT?

Reply:
I think we log this already, the third field in the `build[{}/{}/{}/{}]` bit is `Build.CURRENT.hash()`.